x-pack/plugins/ml/public/application/model_management/models_list.tsx

```diff
@@ -653,102 +611,110 @@ export const ModelsList: FC<Props> = ({
             />
           ) : null;
 
-        return tooltipContent ? (
-          <EuiToolTip content={tooltipContent}>
-            <>
-              {descriptionText}&nbsp;
-              <EuiIcon type={'warning'} color="warning" />
-            </>
-          </EuiToolTip>
-        ) : (
-          descriptionText
+        return (
+          <EuiFlexGroup gutterSize={'xs'} direction={'column'}>
+            <EuiFlexGroup gutterSize={'s'} alignItems={'center'} wrap={true}>
+              <EuiFlexItem grow={false}>
+                <strong data-test-subj="mlModelsTableColumnIdValueId">{modelId}</strong>
+              </EuiFlexItem>
+              {isTechPreview ? (
+                <EuiFlexItem grow={false}>
+                  <TechnicalPreviewBadge compressed />
+                </EuiFlexItem>
+              ) : null}
+            </EuiFlexGroup>
+
+            {descriptionText ? (
+              <EuiText
+                color={'subdued'}
+                size={'xs'}
+                data-test-subj="mlModelsTableColumnIdValueDescription"
+              >
+                {tooltipContent ? (
+                  <EuiToolTip content={tooltipContent}>
```

Comment:
Can this tooltip be attached to the icon rather than the text? Currently it can be rendered quite a long way from the warning text?

<img width="1822" alt="Screenshot 2024-10-07 at 17 53 39" src="https://github.com/user-attachments/assets/49b10970-8fc4-49c0-9aa0-1a2a51750caa">

Reply:
Updated in https://github.com/elastic/kibana/pull/194614/commits/aa9db4a63fb3bba6536dce02336b5b84986cdecb